ocsptools/ocsp_tools.go

```diff
@@ -1,16 +1,21 @@
 package ocsptools
 
 import (
-	"../linter"
 	"bytes"
 	"crypto"
 	"crypto/x509"
 	"encoding/base64"
 	"fmt"
+	"github.com/googleinterns/ocsp-response-linter/linter"
```

Comment:
Same comment here.

Reply:
I learned from talking with Filippo today that using relative paths is not a good practice and instead I should be using urls like this.